src/compas_viewer/scene/bufferobject.py

```diff
@@ -206,7 +207,8 @@ def __init__(
         self.linewidth = 1.0 if linewidth is None else linewidth
         self.opacity = 1.0 if opacity is None else opacity
         self.doublesided = True if doublesided is None else doublesided
-        self.is_visible = True if is_visiable is None else is_visiable
+        self.show = True if is_visiable is None else is_visiable
```

Comment:
`is_visiable` => `is_visible`

Reply:
btw, i am not a fan of `self.show`. i think it should be `self.is_visible`. `self.show` sounds like a method...

Reply:
`show` comes from compas core, which I suggested a while ago for the simplicity, but I agree indeed it can feel like a method. If we decide to go back to `is_visible`, I would suggest we make this change in compas core first then update here